src/Couchbase.TravelSample/Models/RouteCreateRequestCommand.cs

```diff
@@ -0,0 +1,46 @@
+using System.Text.Json.Serialization;
+
+
+namespace Couchbase.TravelSample.Models;
+
+public record RouteCreateRequestCommand
+{
+    [JsonPropertyName("airline")]
+    public string Airline { get; set; } = string.Empty;
+
+    [JsonPropertyName("airlineid")]
+    public string AirlineId { get; set; } = string.Empty;
+
+    [JsonPropertyName("destinationairport")]
+    public string DestinationAirport { get; set; } = string.Empty;
+
+    [JsonPropertyName("distance")]
+    public double Distance { get; set; }
+
+    [JsonPropertyName("equipment")]
+    public string Equipment { get; set; } = string.Empty;
+
+    [JsonPropertyName("schedule")]
+    public List<Schedule>? Schedule { get; set; }
```

Comment:
Is a concrete implementation something we want in a command?  Will this collection have information added or removed?

Reply:
Yes there will be multiple entries of Schedule which will be added or removed.